css/index.css

```diff
@@ -112,35 +178,87 @@ p {
     justify-content: center;
 }
 
-.bottom-content {
-    display: flex;
-    margin: 0 2% 20px;
-    justify-content: space-around;
+nav a:hover {
+    text-decoration: underline;
 }
 
-.bottom-content .text-container {
-    padding-right: 4%;
+p {
+    line-height: 1.4;
 }
 
-.bottom-content .text-container:last-child {
-    padding-right: 0;
+.pseudo-button {
+    border: 2px solid black;
+    border-radius: 10px;
+    display: inline-block;
+    padding: .6em 1.5em;
+    margin: 1.25em 0;
 }
 
-footer {
-    width: 100%;
-    background: black;
+.top-about {
+    margin-bottom: 20px;
+    border-bottom: 1px dashed black;
+    padding-bottom: 20px;
 }
 
-footer nav {
-    width: 60%;
+.top-about h2, .top-about p {
+    padding: 0 1em;
+}
+
+.top-content {
     display: flex;
-    justify-content: space-between;
-    align-items: center;
-    padding: 20px 2%;
-    font-size: 14px;
+    flex-wrap: wrap;
+    justify-content: space-evenly;
+    margin-bottom: 20px;
+    border-bottom: 1px dashed black;
 }
 
-footer nav a {
-    color: white;
-    text-decoration: none;
+.top-content .text-container {
+    width: 48%;
+    padding: 0 1%;
+    padding-bottom: 20px;  
+}
+
+/* Let's break some stuff! */
+/* Mobile responsive stretch attempts below */
+
+@media only screen and (min-width: 26rem) {
+    body {
```

Comment:
Love these attempts as a mobile view. By the end of this week, you should feel pretty comfortable at this!